packages/chrome/src/ChromeProvider/ChromeProvider.test.tsx

```diff
@@ -31,17 +32,39 @@ describe('ChromeProvider', () => {
     expect(getSpy).toHaveBeenCalledWith('/api/chrome-service/v1/user');
   });
 
-  test('should post new data on pathname change', async () => {
+  test('should post new data on title change', async () => {
     getSpy.mockResolvedValueOnce([]);
     postSpy.mockResolvedValue(['/', '/bar']);
+    const DocumentMutator = () => {
+      useEffect(() => {
+        document.title = 'Foo title';
+      }, []);
+      return null;
+    };
+    // mock the initial document title
+    document.title = 'Initial title';
     await act(async () => {
       render(
-        <MemoryRouter initialEntries={['/']} initialIndex={0}>
-          <Routes>
-            <Route path="*" element={<Link to="/foo/bar">/foo/bar</Link>}></Route>
-          </Routes>
-          <ChromeProvider bundle="bundle-title" />
-        </MemoryRouter>
+        <ScalprumContext.Provider
+          // @ts-ignore
+          value={{
```

Comment:
NIT - adding ts-ignore here may prevent us from catching breaking API changes in the future. It might be nice to just pass in the 3 missing values:

```ts
          value={{
            initialized: false,
            config: {},
            pluginStore: new PluginStore(),
            api: {
              chrome: {
                getBundleData: () => ({
                  bundleTitle: 'bundle-title',
                }),
              },
            },
          }}
```